README.md

```diff
@@ -51,27 +51,41 @@ Once the implementation of a piece of functionality is considered to be bug
 free and properly documented (both API docs and an example script),
 it can be incorporated into the master branch.
 
-To run the tests using `pytest`, you need to have the cloned mne-python repository
-with a local pip-install instead of the mne-python package from pypi. Update your
-installation as follows:
+To help developing mne-bids, you will need a few adjustments to your
+installation as shown below.
 
-    $ git clone https://github.com/mne-tools/mne-python
+##### Running tests
+
+To run the tests using `pytest`, you need to have the git cloned mne-python
+repository with a local pip-install instead of the mne-python package from
+pypi. Update your installation as follows.
+
+    $ git clone https://github.com/mne-tools/mne-python --depth 1
     $ cd mne-python
     $ pip uninstall mne  # uninstall pypi mne
     $ pip install -e .  # use the cloned repo for a local install of mne
+    $ pip install flake8 pytest pytest-cov pytest-sugar pytest-faulthandler
```

Comment:
you don't need `pytest-sugar` and `pytest-faulthandler` do you?

Reply:
also this looks like it's part of the instructions to install dev version of `mne`

Reply:
no, they are not strictly needed ... they just make the output look nicer. I can remove them from the README if you like ... but i would leave them in our own CIs

Reply:
yes +1 to remove it from README but keep it in CI ...